install.sh

```diff
@@ -15,7 +15,7 @@ latest_version() {
 }
 
 repo_url() {
-  echo "https://github.com/duniter/duniter.git"
+  echo "https://github.com/blavenie/duniter.git"
```

Comment:
You could:
- add a new commit to revert that commit.
- `drop` that commit using [`git rebase`](https://stackoverflow.com/questions/1186535/how-to-modify-a-specified-commit-in-git).

Modifications done on your branch will automatically be updated here.